Foundation/NSNotificationQueue.swift

```diff
@@ -109,16 +109,16 @@ open class NotificationQueue: NSObject {
         var predicate: (NSNotificationListEntry) -> Bool
         switch coalesceMask {
         case [.onName, .onSender]:
-            predicate = { (entryNotification, _) in
-                return _SwiftValue.store(notification.object) !== _SwiftValue.store(entryNotification.object) || notification.name != entryNotification.name
+            predicate = { entry in
```

Comment:
Unfortunately these three still gave the error 'does not support destructuring' even after I checked it with the latest compiler, but it was the only ones.